src/main/scala/uk/gov/nationalarchives/Lambda.scala

```diff
@@ -80,7 +80,7 @@ class Lambda extends RequestHandler[SQSEvent, Unit] {
         _ <- s3
           .copy(outputBucket, metadataFileInfo.id.toString, outputBucket, s"$batchRef/data/${metadataFileInfo.id}")
         _ <- s3.deleteObjects(outputBucket, fileNameToFileInfo.values.map(_.id.toString).toList)
-        _ <- sfn.startExecution(config.sfnArn, output, Option(s"$batchRef-${randomUuidGenerator()}"))
+        _ <- sfn.startExecution(config.sfnArn, output, Option(s"$batchRef"))
```

Comment:
You can just do `Option(batchRef)`

Reply:
Ah, I can't believe I missed that. Thanks, I've fixed it.